packages/core/src/application.ts

```diff
@@ -61,9 +62,95 @@ export class Application extends Context {
    */
   controller(controllerCtor: ControllerClass, name?: string): Binding {
     name = name || controllerCtor.name;
-    return this.bind(`controllers.${name}`)
+    return this.bind(`${CoreBindings.CONTROLLERS}.${name}`)
       .toClass(controllerCtor)
-      .tag('controller');
+      .tag(CoreBindings.CONTROLLERS_TAG);
+  }
+
+  /**
+   * Register a booter class with this application.
+   *
+   * @param booterCls {Function} The booter class (constructor function).
+   * @param {string=} name Optional booter name, defaults to the class name.
+   * @return {Binding} The newly created binding, you can use the reference to
+   * further modify the binding, e.g. lock the value to prevent further
+   * modifications.
+   *
+   * ```ts
+   * class MyBooter implements Booter {}
+   * app.booter(MyBooter);
+   * ```
+   */
+  booter<T extends Booter>(booterCls: Constructor<T>, name?: string): Binding {
+    name = name || booterCls.name;
+    return this.bind(`${CoreBindings.BOOTERS}.${name}`)
+      .toClass(booterCls)
+      .tag(CoreBindings.BOOTERS_TAG)
+      .inScope(BindingScope.SINGLETON);
+  }
+
+  /**
+   * Register an array of booter classes with this application.
+   * Each Booter added in this way will automatically be named based on the
+   * class constructor name with the "booters." prefix.
+   *
+   * If you wish to control the binding keys for particular booter instances,
+   * use the app.booter function instead.
+   *
+   * @param {Constructor<Booter>[]} booterArr {Function} An array of Booter
+   * constructors.
+   * @return {Binding[]} An array of bindings for the registered Booter classes.
+   *
+   * ```ts
+   * app.booters([ControllerBooter, RepositoryBooter]);
+   * ```
+   */
+  booters<T extends Booter>(booterArr: Constructor<T>[]): Binding[] {
```

Comment:
I strongly dislike when we have two entities with almost the same name, differing only in the `s` suffix. Such difference is very easy to overlook when reading code. Please use a more distinctive names, or perhaps allow `app.booter` to accept both a single booter or an array of booters.